dev/com.ibm.ws.messaging.open_jms20_fat/bnd.bnd

```diff
@@ -0,0 +1,54 @@
+#*******************************************************************************
```

Comment:
Update the copyright here.  Also, I noticed this copyright inconsistency in a few other files.